languages/csharp/Bitwarden.Sdk/ProjectsClient.cs

```diff
@@ -0,0 +1,74 @@
+﻿namespace Bitwarden.Sdk;
+
+public class ProjectsClient
+{
+    private readonly CommandRunner _commandRunner;
+
+    internal ProjectsClient(CommandRunner commandRunner)
+    {
+        _commandRunner = commandRunner;
+    }
+
+    public ProjectResponse Get(Guid id)
+    {
+        var command = new Command() { Projects = new ProjectsCommand { Get = new ProjectGetRequest { Id = id } } };
+        var result = _commandRunner.RunCommand<ResponseForProjectResponse>(command);
+
+        if (result is { Success: true }) return result.Data;
+        throw new BitwardenException(result != null ? result.ErrorMessage : "Project not found");
```

Comment:
Command runner is to generic, I wouldn't add it there. Also I have this default messages if result is null. Also creating some private method could be an option but it would be the same amount of repetitions